### PR TITLE
Use source svg files instead of exported pngs

### PR DESCRIPTION
**Pull request by amirebrahimi**
## Description

All major browsers for the past 10 years have  [_basic_ SVG support](https://caniuse.com/?search=svg). When generating the workflow image for REM it required @andreamari to dig it up. I was thinking that it may be good to have it checked in to the repository, so that new figures can be created easily, but then just realized we could use it directly in place of the png. It's a smaller file size, too.

---

### License

- [X] I license this contribution under the terms of the GNU GPL, version 3 and grant Unitary Fund the right to provide additional permissions as described in section 7 of the GNU GPL, version 3.

Before opening the PR, please ensure you have completed the following where appropriate.
- [ ] I added unit tests for new code.
- [ ] I used [type hints](https://www.python.org/dev/peps/pep-0484/) in function signatures.
- [ ] I used [Google-style](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods) docstrings for functions.
- [ ] I [updated the documentation](../blob/master/docs/CONTRIBUTING_DOCS.md) where relevant.
- [ ] Added myself / the copyright holder to the AUTHORS file

### CODE REVIEW

File: docs/source/img/rem_workflow.svg [hunk 1]
```diff
@@ -0,0 +1,383 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (natestemen) [inline docs/source/img/rem_workflow.svg:1]: I'm not sure what happened, but it looks like the arrows are missing from this diagram.

Comment (amirebrahimi) [reply docs/source/img/rem_workflow.svg:1]: I'm seeing arrows here:
Screen Shot 2022-12-26 at 12 40 09 PM

You're not?

Comment (natestemen) [reply docs/source/img/rem_workflow.svg:1]: Strangely, no!

Screenshot 2022-12-26 at 11 55 33 PM

After testing on Chromium it seems to work there, but not on firefox. What browser are you on?

Comment (amirebrahimi) [reply docs/source/img/rem_workflow.svg:1]: I'm on Brave, which is Chromium. Do you have the latest Firefox version?

Comment (amirebrahimi) [reply docs/source/img/rem_workflow.svg:1]: I tried with Firefox 72.0.2 on OSX and it rendered correctly:
Screen Shot 2022-12-27 at 7 50 03 PM

Comment (WrathfulSpatula) [reply docs/source/img/rem_workflow.svg:1]: Firefox 108.0.1 (64-bit) on Ubuntu 22.04 LTS doesn't render arrows, either, rem_workflow.svg

Comment (natestemen) [reply docs/source/img/rem_workflow.svg:1]: Thanks for checking Dan. I'm also on Firefox 108.0 (64-bit) on macOS Ventura 13.0.

Amir, if you upgrade firefox does it still work?

Comment (amirebrahimi) [reply docs/source/img/rem_workflow.svg:1]: With Firefox 108.0.2 it does on macOS Catalina 10.15.7 it doesn't render the arrows. I'll see if I can track down why.

Comment (amirebrahimi) [reply docs/source/img/rem_workflow.svg:1]: Should be fixed now. The trick is to use "Stroke to Path" feature in Inkscape, which splits the line and arrow out to separate shapes. I kept the original line/arrow in there, too, as a source and just hid the layer.

Comment (amirebrahimi) [reply docs/source/img/rem_workflow.svg:1]: image

**Comment by andreamari** [comment]
I had the same "_arrow problem_" too. But now it works!

**Comment by amirebrahimi** [comment]
I think the failed test is unrelated. I can push an empty commit if you'd like to see it re-run.

**Comment by natestemen** [comment]
I've re-run the failed action.

**Review by natestemen — APPROVED** [review]
Once the build has passed this looks good! Thanks Amir for updating these!

**Pull request merged**